pkg/agent/attestor/workload/workload.go

```diff
@@ -43,7 +43,7 @@ const (
 // Attest invokes all workload attestor plugins against the provided PID. If an error
 // is encountered, it is logged and selectors from the failing plugin are discarded.
 func (wla *attestor) Attest(ctx context.Context, pid int32) []*common.Selector {
-	tLabels := []telemetry.Label{{workloadPid, string(pid)}}
+	tLabels := []telemetry.Label{{workloadPid, fmt.Sprint(pid)}}
```

Comment:
we should get rid of pid labels based on evan's comments

Reply:
yea, I will remove them